client/src/widgets/all-houses/houses-performance-analytic/HousesPerformanceTable.tsx

```
@@ -27,7 +35,23 @@ export const HousesPerformanceTable = <T,>({ table, limit, setLimit }: Props<T>)
+                tableGrid,
+                'cursor-pointer transition-colors duration-300 ease-out hover:bg-muted-foreground text-text',
+              )}
+              onClick={() => handleRouteToRenter(row.original.id)}
+              onKeyDown={e => {
+                if (e.key === 'Enter' || e.key === ' ') {
+                  e.preventDefault();
+                  handleRouteToRenter(row.original.id);
+                }
+              }}
+              tabIndex={0}
+              role="button"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)